sources/api/apiserver/src/server/ephemeral_storage.rs

```diff
@@ -297,6 +297,7 @@ pub fn allowed_bind_dirs(variant: &str) -> BindDirs {
     if variant.contains("k8s") {
         allowed_exact.insert("/var/lib/kubelet");
         allowed_exact.insert("/var/log/pods");
+        allowed_exact.insert("/var/lib/soci-snapshotter-grpc");
```

Comment:
The `-grpc` suffix is kind of ugly so if it's trivial to reconfigure this path to be `/var/lib/soci-snapshotter` then we should do that. Otherwise LGTM.

Reply:
Should be trivial, just have to shuffle around some changes to account for it, but in service startup I'll modify the snapshotter root to be `/var/lib/soci-snapshotter`:
```
  -root string
    	path to the root directory for this snapshotter (default "/var/lib/soci-snapshotter-grpc/")
```

Reply:
Updated to `/var/lib/soci-snapshotter` See https://github.com/bottlerocket-os/bottlerocket-core-kit/pull/569/commits/7e90f546ff03a3aba1647348122d15b98602b147 from #569 for corresponding change in the service